.github/workflows/deploy.yml

```diff
@@ -0,0 +1,93 @@
+on:
+  workflow_call:
+    inputs:
+      image_version:
+        required: false
+        type: string
+        description: "Docker image version to pull for deployment"
+        default: "latest"
+      branch:
+        required: false
+        description: "Branch to pull in instance deployment"
+        type: string
+        default: "main"
+      volume_size:
+        required: false
+        description: "Size in GB of instance"
+        type: number
+        default: 10
```

Comment:
If it changes the price we can lower up to 5 GB (currently backend image is ~2GB and frontend ~30MB)

Reply:
Is it necessary to repeat te default for variables like volume_size, branch, since they are already in `cd/terraform/variable.tf` ?

Reply:
I don't know how is the pricing for volume boot instance. 

For defaults values, is more documentation than real defaults values.